index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Editor</title>
+    
+    <style type="text/css" media="screen">
+      body {
+          overflow: hidden;
+      }
+
+      #editor {
+          margin: 0;
+          position: absolute;
+          top: 0;
+          bottom: 0;
+          left: 0;
+          right: 0;
+      }
+    </style>
+
+  </head>
+
+  <body>
+
+    <pre id="editor">
+      * {
+        background-color: blue;
+        color: orange;
+      }
+    </pre>
+
+    <!-- load ace -->
+    <script src="src-noconflict/ace.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
Ace is simple. Better than codemirror. The documentation is confusing. Awesome!

Reply:
It was. I spent way too much time trying to get Ace set up but once I figured it out, I think it is pretty cool.